src/backend/nodes/data_ingestion.py

```
@@ -0,0 +1,163 @@
+                    except UnicodeDecodeError:
+                        data[table_name] = pd.read_csv(filepath, encoding="latin-1")
+                except (pd.errors.ParserError, UnicodeDecodeError) as exc:
+                    print(f"Error parsing CSV file {filename}: {exc}")
+                except FileNotFoundError as exc:
+                    print(f"File not found {filename}: {exc}")
+                except Exception as exc:  # noqa: BLE001
+                    print(f"Unexpected error loading {filename}: {exc}")
+
+        return data
+
+    def post(self, shared, prep_res, exec_res):
```

Ruff (0.14.10)

Unused method argument: `prep_res`

(ARG002)